src/txn/txn_log.c

```diff
@@ -240,64 +231,62 @@ err:		__wt_logrec_free(session, &logrec);
 int
 __wt_txn_log_op(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt)
 {
-	WT_CONNECTION_IMPL *conn;
-	WT_DECL_RET;
-	WT_ITEM *logrec;
-	WT_TXN *txn;
-	WT_TXN_OP *op;
-
-	uint32_t fileid;
-
-	conn = S2C(session);
-	txn = &session->txn;
-
-	if (!FLD_ISSET(conn->log_flags, WT_CONN_LOG_ENABLED) ||
-	    F_ISSET(session, WT_SESSION_NO_LOGGING) ||
-	    (F_ISSET(S2BT(session), WT_BTREE_NO_LOGGING) &&
-	    !FLD_ISSET(conn->log_flags, WT_CONN_LOG_DEBUG_MODE)))
-		return (0);
-
-	/* We'd better have a transaction. */
-	WT_ASSERT(session,
-	    F_ISSET(txn, WT_TXN_RUNNING) && F_ISSET(txn, WT_TXN_HAS_ID));
-
-	WT_ASSERT(session, txn->mod_count > 0);
-	op = txn->mod + txn->mod_count - 1;
-	fileid = op->btree->id;
-
-	/*
-	 * If this operation is diagnostic only, set the ignore bit on the
-	 * fileid so that recovery can skip it.
-	 */
-	if (F_ISSET(S2BT(session), WT_BTREE_NO_LOGGING) &&
-	    FLD_ISSET(conn->log_flags, WT_CONN_LOG_DEBUG_MODE))
-		FLD_SET(fileid, WT_LOGOP_IGNORE);
-
-	WT_RET(__txn_logrec_init(session));
-	logrec = txn->logrec;
-
-	switch (op->type) {
-	case WT_TXN_OP_NONE:
-	case WT_TXN_OP_INMEM_COL:
-	case WT_TXN_OP_INMEM_ROW:
-	case WT_TXN_OP_REF_DELETE:
-		/* Nothing to log, we're done. */
-		break;
-	case WT_TXN_OP_BASIC_COL:
-	case WT_TXN_OP_BASIC_ROW:
-		ret = __txn_op_log(session, logrec, op, cbt, fileid);
-		break;
-	case WT_TXN_OP_TRUNCATE_COL:
-		ret = __wt_logop_col_truncate_pack(session, logrec, fileid,
-		    op->u.truncate_col.start, op->u.truncate_col.stop);
-		break;
-	case WT_TXN_OP_TRUNCATE_ROW:
-		ret = __wt_logop_row_truncate_pack(session, logrec, fileid,
-		    &op->u.truncate_row.start, &op->u.truncate_row.stop,
-		    (uint32_t)op->u.truncate_row.mode);
-		break;
-	}
-	return (ret);
+    WT_CONNECTION_IMPL *conn;
+    WT_DECL_RET;
+    WT_ITEM *logrec;
+    WT_TXN *txn;
+    WT_TXN_OP *op;
+
+    uint32_t fileid;
+
+    conn = S2C(session);
+    txn = &session->txn;
+
+    if (!FLD_ISSET(conn->log_flags, WT_CONN_LOG_ENABLED) ||
+      F_ISSET(session, WT_SESSION_NO_LOGGING) ||
+      (F_ISSET(S2BT(session), WT_BTREE_NO_LOGGING) &&
+        !FLD_ISSET(conn->log_flags, WT_CONN_LOG_DEBUG_MODE)))
```

Comment:
This if looks a bit weird with the 3rd line of re-indent being extra indented

Reply:
As discussed before, this is default behaviour for line breaks within a nested expression. Related LLVM discussion here: https://reviews.llvm.org/D42787